### PR TITLE
COMMERCE-8027 Save and Import buttons disabled if no selection provided

### PR DESCRIPTION
**Pull request by upalatucci**
@matijapetanjek @FabioDiegoMastrorilli

### CODE REVIEW

**Comment by upalatucci** [comment]
ci:test:sf

File: modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/js/FileParsers.js [hunk 1]
```diff
@@ -74,6 +83,7 @@ function parseInChunk({chunkParser, file, onComplete, onError, options}) {
 	const chunkReaderBlock = (_offset, length, _file) => {
 		const reader = new FileReader();
 		const blob = _file.slice(_offset, length + _offset);
```
Comment (FabioDiegoMastrorilli) [inline modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/js/FileParsers.js:85]: const blog to be defined before :)

File: modules/apps/batch-planner/batch-planner-web/test/components/Import/ImportForm.js [hunk 2]
```diff
@@ -78,6 +78,24 @@ describe('ImportForm', () => {
...
+		const {getByText} = render(<ImportForm {...BASE_PROPS} />);
+
+		act(() =>
+			Liferay.fire(SCHEMA_SELECTED_EVENT, {
+				schema: SCHEMA,
+			})
+		);
+
+		act(() =>
+			Liferay.fire(FILE_SCHEMA_EVENT, {
```
Comment (FabioDiegoMastrorilli) [inline modules/apps/batch-planner/batch-planner-web/test/components/Import/ImportForm.js:91]: You can use the same `act` for both the `Liferay.fire`

**Review by FabioDiegoMastrorilli — CHANGES_REQUESTED** [review]
Brilliant! Lots of spaces :D 
Just 1 super small thing to be changed and another trivial optimization

**Comment by upalatucci** [comment]
Not enough !! :laughing: 

Thanks for the review

**Comment by upalatucci** [comment]
ci:forward

**Pull request closed without merging**